tests/suites/test_suite_ssl.function

```diff
@@ -339,6 +341,230 @@ int mbedtls_mock_tcp_recv_nb( void *ctx, unsigned char *buf, size_t len )
     return mbedtls_test_buffer_get( socket->input, buf, len );
 }
 
+
+/*
+ * Structure with endpoint certificates for SSL comunication tests.
+ */
+typedef struct mbedtls_endpoint_certificate{
+    mbedtls_x509_crt ca_cert;
+    mbedtls_x509_crt cert;
+    mbedtls_x509_crt cert2;
+    mbedtls_pk_context pkey;
+    mbedtls_pk_context pkey2;
+} mbedtls_endpoint_certificate;
+
+/*
+ * Endpoint structure for SSL comunication tests.
+ */
+typedef struct mbedtls_endpoint{
```

Comment:
Please follow the same coding standards as other struct typedefs in this file by moving the opening bracket to the next line.

Reply:
done